css/selectors/contains-invalid.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Selectors: selector list containing invalid selector</title>
+    <link rel="help" href="https://drafts.csswg.org/selectors-4/#grouping">
+    <meta name="assert" content="If a selector is invalid, the entire selector list is invalid.">
+    <meta name="assert" content="Pseudo-elements are not valid within :is :not :where.">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <style>
+      .a, .b, .c {
+        color: lime;
+      }
+
+      .a, :nonsense {
+        color: red;
+      }
+
+      /* Pseudo-elements cannot be represented by the matches-any pseudo-class; they are not valid within :is(). */
+      .b, :is(::before) {
```

Comment:
As the edits for https://github.com/w3c/csswg-drafts/issues/3264#issuecomment-440752606 haven't been done yet and the discussion is still ongoing, it's not entirely clear what should happen to `:is()` with a single invalid argument. I would expect that the new invalidation rules would mean that since `::before` is not allowed in `:is()` it gets discarded, but that this merely cause that argument of `:is()` (in this case its only argument) to be discarded, and the `:is()` construct as a whole remains valid, even if it matches nothing. So the whole rule is not treated as a syntax error, `.b` matches, and the element turns red. Which makes the test wrong.